schema/new_library_verification_cases_schema.json

```diff
@@ -0,0 +1,3604 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+        "cases": {
+            "type": "array",
+            "items": [
+                {
+                    "type": "object",
+                    "properties": {
+                        "no": {
+                            "type": "integer"
+                        },
+                        "run_simulation": {
+                            "type": "boolean"
+                        },
+                        "simulation_IO": {
+                            "type": "object",
+                            "properties": {
+                                "idf": {
+                                    "type": "string"
+                                },
+                                "idd": {
+                                    "type": "string"
+                                },
+                                "weather": {
+                                    "type": "string"
+                                },
+                                "output": {
+                                    "type": "string"
+                                },
+                                "ep_path": {
+                                    "type": "string"
+                                }
+                            },
+                            "required": [
+                                "idf",
+                                "idd",
+                                "weather",
+                                "output",
+                                "ep_path"
+                            ]
+                        },
```

Comment:
There may need to be conditional logic here to only require "idd", "weather", and "ep_path" when "run_simulation" is True. I'd check this to make sure, but this is what I found when working on the supply air temp reset measure